core/amber/src/main/python/core/runnables/data_processor.py

```diff
@@ -112,7 +112,7 @@ def _report_exception(self, exc_info: ExceptionInfo):
         self._context.console_message_manager.put_message(
             ConsoleMessage(
                 worker_id=self._context.worker_id,
-                timestamp=datetime.now(),
+                timestamp=datetime.datetime.now(datetime.timezone.utc),
```

Comment:
if we are using utc time here, what would be the timezone shown on the frontend? note that our services currently mostly deploy in PDT time, which is different from UTC.

If the frontend shows a different timezone, can we use the server's timezone to send all console messages? For this purpose, you may want to check out pytz or tzlocal. See here for more https://stackoverflow.com/questions/13218506/how-to-get-system-timezone-setting-and-pass-it-to-pytz-timezone